.github/workflows/ci-test-backend.yml

```diff
@@ -51,21 +53,17 @@ jobs:
           go get -u github.com/mattn/goveralls
 
       - name: run backend linters
+        run: $GITHUB_WORKSPACE/golangci-lint run --config .golangci.yml ./...
+        working-directory: backend
+
+      - name: run linters for examples
         run: |
```

Comment:
This one can be converted to single line as well.

Reply:
ready